server/inbox/migrations/0004_auto_20210219_2025.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.1.6 on 2021-02-19 20:25
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('inbox', '0003_auto_20210219_0515'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='inbox',
+            name='author',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```

Comment:
Just a nit: I would delete all four migration files and regenerate the file so that only one file is pushed ;)

Reply:
Didn't know you could do that, thanks!